### PR TITLE
Add Interim Treasury Program Info

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR updates the treasury page to include changes coming with the Interim Treasury Program

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

N/A

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [x] No additional PRs are required after the translations are done

#### Items to be Updated

N/A

### CODE REVIEW

File: learn/features/treasury.md [hunk 1]
```diff
@@ -9,66 +9,65 @@ description: As a Polkadot parachain, Moonbeam has an on-chain treasury controll
...
-    |      Proposal bond minimum       |                                   {{ networks.moonbase.treasury.proposal_bond_min }} DEV                                    |
-    |      Proposal bond maximum       |                                     {{ networks.moonbase.treasury.proposal_bond_max }}                                      |
-    |         Motion duration          | {{ networks.moonbase.treasury.motion_duration_blocks }} blocks ({{ networks.moonbase.treasury.motion_duration_days }} days) |
-    |           Spend period           |    {{ networks.moonbase.treasury.spend_period_blocks }} blocks ({{ networks.moonbase.treasury.spend_period_days}} days)     |
-    |    Maximum approved proposals    |                                   {{ networks.moonbase.treasury.max_approved_proposals }}                                   |
-    | % of transaction fees allocated  |                                     {{ networks.moonbase.treasury.tx_fees_allocated }}                                      |
+    |                      Variable                      |                                                            Value                                                            |
```
Comment (albertov19) [inline learn/features/treasury.md:54]: Moonbase Alpha has 2 members in the Treasury Collective :)

File: learn/features/treasury.md [hunk 2]
```diff
@@ -9,66 +9,65 @@ description: As a Polkadot parachain, Moonbeam has an on-chain treasury controll
...
+    |               Proposal bond minimum                |                                   {{ networks.moonbase.treasury.proposal_bond_min }} DEV                                    |
+    |               Proposal bond maximum                |                                     {{ networks.moonbase.treasury.proposal_bond_max }}                                      |
+    |                  Motion duration                   | {{ networks.moonbase.treasury.motion_duration_blocks }} blocks ({{ networks.moonbase.treasury.motion_duration_days }} days) |
+    |                    Spend period                    |    {{ networks.moonbase.treasury.spend_period_blocks }} blocks ({{ networks.moonbase.treasury.spend_period_days}} days)     |
+    |             Maximum approved proposals             |                                   {{ networks.moonbase.treasury.max_approved_proposals }}                                   |
+    | Target % of transaction fees allocated to Treasury |                                     {{ networks.moonbase.treasury.tx_fees_allocated }}                                      |
 
 ## Community Treasury {: #community-treasury }
```
Comment (albertov19) [inline learn/features/treasury.md:64]: Should we include anything around the Forum? There is also the optional IDEA step and the mandatory forum post discussion

Comment (eshaben) [reply learn/features/treasury.md:64]: There was a sentence added in the following paragraph with a link to the forum

**Review by albertov19 — APPROVED** [review]
LGTM!

**Pull request merged**